scripts/bungie-data-helper.coffee

```diff
@@ -1,13 +1,47 @@
 request = require('request')
 
 class DataHelper
+  statHashes = 
```

Comment:
Good point.  
For one, they are constant (per Manifest); I was also following your example with the weapon buckets and some recommendation I read in regard to caching the DestinyStatDefinitions from the Manifest, but I see no reason, other than lookup speed, to use constants.

Let me know if you'd like me to change it to look up the `definitions.stats` instead.